### ISSUE TITLE
prototype.registerCallback is not a function

### ISSUE THREAD

**Issue by dprice000** [sup3rb0wlz/polymer-nb-menu#2]
I keep getting an error on line 74 of polymer-micro.html.

**Comment by NickBolles** [comment]
This was made for polymer v0.5. I haven't looked at polymer v1.0 yet. It probly needs a complete rehaul for v1.0.

**Comment by dprice000** [comment]
Thanks for the quick response. I thought of that after I created the issue. I am now trying to run the tool using the bower package.

**Comment by maxwowpow** [comment]
Does it make sense to update to Polymer 1, since it's stable and folks can use the component more intensively?

**Comment by NickBolles** [comment]
Yes it does. But I have not been using Polymer for a long time and from what little research I have done it would be quite a bit of re-learning. If someone would like to take a stab at it and submit a PR that would be awesome. But I won't have time for a while to look at it unfortunately.
